Small fix to get the proof working again. Removed lsu_data_i from the DTI code.

tfmelham opened the pull request:
A recent commit removed the unused input signal lsu_rdata_i, which was included in the DTI conditions of the proof. The condition isn't needed and has been removed. And the proof runs again.

marnovandermaas reviewed (commented):
Looks good to me, would you mind squashing these two commits?
You can do this by typing:
`git rebase -i HEAD~2`
Then in text file that comes up, in front of the second commit change the word "pick" to "squash" and write the file.

tfmelham commented:
Okay - I will try that! (When I get back to my PC.)

Thanks!
Tom

On 30 May 2025, at 12:45, Marno van der Maas ***@***.***> wrote:

﻿

@marnovandermaas commented on this pull request.

Looks good to me, would you mind squashing these two commits?
You can do this by typing:
git rebase -i HEAD~2
Then in text file that comes up, in front of the second commit change the word "pick" to "squash" and write the file.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/cheriot-ibex/pull/84#pullrequestreview-2881106024>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AALI4BURSGSTKOT4UQHW6SD3BA77JAVCNFSM6AAAAAB6HYG2ZGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQOBRGEYDMMBSGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

tfmelham commented:
I get this error message:

> git pull --tags origin fvregress
From github.com:tfmelham/cheriot-ibex
* branch              fvregress  -> FETCH_HEAD
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint:
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint:
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.

Tom

From: Thomas Melham ***@***.***>
Date: Friday, 30 May 2025 at 16:53
To: cheriot-ibex microsoft ***@***.***>
Cc: cheriot-ibex microsoft ***@***.***>, Melham Tom ***@***.***>, Author ***@***.***>
Subject: Re: [microsoft/cheriot-ibex] Small fix to get the proof working again. Removed lsu_data_i from the DTI code. (PR #84)
Hmmm… I did this but now appear to be in a weird state.

***@***.***

Tom

From: Thomas Melham ***@***.***>
Date: Friday, 30 May 2025 at 12:53
To: cheriot-ibex microsoft ***@***.***>
Cc: cheriot-ibex microsoft ***@***.***>, Melham Tom ***@***.***>, Author ***@***.***>
Subject: Re: [microsoft/cheriot-ibex] Small fix to get the proof working again. Removed lsu_data_i from the DTI code. (PR #84)
Okay - I will try that! (When I get back to my PC.)

Thanks!
Tom

On 30 May 2025, at 12:45, Marno van der Maas ***@***.***> wrote:
﻿

@marnovandermaas commented on this pull request.

Looks good to me, would you mind squashing these two commits?
You can do this by typing:
git rebase -i HEAD~2
Then in text file that comes up, in front of the second commit change the word "pick" to "squash" and write the file.

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/cheriot-ibex/pull/84#pullrequestreview-2881106024>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AALI4BURSGSTKOT4UQHW6SD3BA77JAVCNFSM6AAAAAB6HYG2ZGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQOBRGEYDMMBSGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

marnovandermaas commented:
After doing the rebase you'll need to overwrite the remote:
`git push --force-with-lease`

kliuMsft commented:
Merging the changes
